lib/mix/lib/mix/utils.ex

```diff
@@ -401,10 +432,14 @@ defmodule Mix.Utils do
     case :file.make_symlink(link, target) do
       :ok ->
         :ok
+
       {:error, _} ->
-        {:ok, File.cp_r!(source, target, fn(orig, dest) ->
-          File.stat!(orig).mtime > File.stat!(dest).mtime
-        end)}
+        {
+          :ok,
+          File.cp_r!(source, target, fn orig, dest ->
+            File.stat!(orig).mtime > File.stat!(dest).mtime
+          end)
+        }
```

Comment:
Please extract to variables and write the tuple on a single line as described in #6643.